### ISSUE TITLE
Format the output for `gty ls`

### ISSUE THREAD

**Issue by ndench** [nicored/gotickyourself#4]
When I run `gty ls` I get output like this:
```
 ~  gty ls 1 week                                                                                                                                                            2088ms  Mon 07 Aug 2017 13:35:43 AEST
2017-08-04 3.7 Administration & Overheads Culture and People 9936342
2017-07-31 1 CSM - DMS General Customer Success 9940060
2017-08-04 2.3 Road Map and Product Development - DMS Development 9936352
2017-08-03 2.7 Road Map and Product Development - DMS Development 9936352
2017-08-02 3 Road Map and Product Development - DMS Development 9936352
2017-07-31 1 Road Map and Product Development - DMS Development 9936352
2017-08-02 1 Road Map and Product Development - DMS Release and Review 9989054
2017-07-31 2 Road Map and Product Development - DMS Release and Review 9989054
2017-08-04 1.4 Road Map and Product Development - DMS Planning and Development Meetings 10009603
2017-08-04 3.5 Road Map and Product Development - DMS Technical Debt 10033034
2017-08-03 3.5 Road Map and Product Development - DMS Technical Debt 10033034
2017-08-02 2 Road Map and Product Development - DMS Technical Debt 10033034
2017-08-01 4 Road Map and Product Development - DMS Technical Debt 10033034
```

Which is pretty hard to decipher. Would be good to group it like:

```
Monday 2017-07-31
1 CSM - DMS General Customer Success 9940060
1 Road Map and Product Development - DMS Development 9936352

Tuesday 2017-08-01
4 Road Map and Product Development - DMS Technical Debt 10033034
```

**Comment by ndench** [comment]
Also, would be good to remove the task ids since they mean nothing to me, and also display the times as `2h 37m` etc. and maybe include the project name too? ie.

```
Monday 2017-07-31
    RedEye
        1h CSM - DMS General Customer Success
        1h Road Map and Product Development - DMS Development

Tuesday 2017-08-01
    RedEye
        4h         Road Map and Product Development - DMS Technical Debt
        1h 30m CSM - DMS General Customer Success
```

**Comment by nicored** [comment]
Done, except for display time which will be solved when resolving issue #10
